src/applications/letters/containers/LetterList.jsx

```diff
@@ -128,50 +128,54 @@ export class LetterList extends React.Component {
         {eligibilityMessage}
 
         <br />
+        <h3 slot="headline">Other sources of VA benefit documentation</h3>
         <p>
           A lot of people come to this page looking for their Post-9/11 GI Bill
           statement of benefits, their Certificate of Eligibility (COE) for home
           loan benefits, and their DD214. We don’t have these documents
           available here yet, but if you’re eligible for them, you can get them
           through these links:
         </p>
-        <ul className="vads-u-margin-bottom--9">
+        <ul
+          className="vads-u-margin-bottom--9"
+          style={{ listStyleType: 'disc' }}
+        >
           <li>
-            <a href="/education/download-letters/" target="_blank">
-              <strong>Download your VA education letters.</strong>
+            <a
+              href="/education/download-letters/"
+              target="_blank"
+              style={{ textDecoration: 'none' }}
+            >
+              <p>VA education letters</p>
```

Comment:
Do these need to be wrapped in `<p>`?  I feel like I typically see text directly in an anchor tag without another paragraph or div wrapping them internally.  I think if we can remove the paragraph tags and it still looks correct, we should do that.